src/modules/logicModule/abstracts/oracleIntegrations/UMA_OptimisticOracleV3/OptimisticOracleIntegrator.sol

```diff
@@ -96,11 +98,11 @@ abstract contract OptimisticOracleIntegrator is
     // Setter Functions
 
     /// @inheritdoc IOptimisticOracleIntegrator
-    function setDefaultCurrency(address _newCurrency)
+    function setDefaultCurrency(address _newCurrency, uint _newBond)
```

Comment:
Not strictly necessary but could be changed to reflect additional usecase. `setDefaultCurrencyAndBond`

Reply:
Fair enough, changed it